packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -210,6 +212,15 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
+  "chain.stateArchiveMode": {
+    hidden: true,
+    choices: Object.values(StateArchiveMode),
+    description: "Strategy to manage archive states",
```

Comment:
want to be more clear about the description

```suggestion
    description: `Strategy to manage archive states, only support ${DEFAULT_STATE_ARCHIVE_MODE} at this time`,
```